### DISCUSSION TITLE
Issues piping aws logs tail --follow to another command.

### DISCUSSION THREAD

**Discussion by EliasToftHansen** [aws/aws-cli#7804] [General]
I would like to pipe the output of 'aws logs tail' with the --follow option to another command, but it doesn't seem to work as I intended.

If I do something like
aws logs tail --follow | cat

I would expect to see more or less the same output as without the pipe to cat, but I see nothing. Not sure what the issue is. I usually pipe stuff around from tail -f etc without issues.

**Comment by EliasToftHansen** [comment]
For anyone interrested, aws logs tail with the --follow option sends log lines to STDERR, so to pipe the loglines into another app you must use |&.

**Reply by garbelini** [reply]
🤦
